exchanges/binanceus/binanceus_types.go

```diff
@@ -524,8 +524,8 @@ type GetTradesParams struct {
 	StartTime  *time.Time `json:"startTime"`
 	EndTime    *time.Time `json:"endTime"`
 	FromID     uint64     `json:"fromId"`
-	Limit      uint       `json:"limit"`
-	RecvWindow uint       `json:"recvWindow"`
+	Limit      uint64     `json:"limit"`
```

Comment:
There are some other instances in this file containing uint and in okx_types.go don't know if you want to change them over as well?